samcli/commands/build/command.py

```diff
@@ -230,6 +243,7 @@ def do_cli(  # pylint: disable=too-many-locals, too-many-statements
     exclude: Optional[Tuple[str, ...]],
     hook_name: Optional[str],
     build_in_source: Optional[bool],
+    mount_with=MountMode.READ.value,
```

Comment:
Default might come from click setting above, so we can add typing and remove default from here as well